src/api/app/policies/assignment_policy.rb

```diff
@@ -4,13 +4,22 @@
 class AssignmentPolicy < ApplicationPolicy
   def create?
     return false unless Flipper.enabled?(:foster_collaboration, user)
-
     return true if user.admin?
+    return false unless assigneer_is_a_collaborator?
 
     record.assignee_is_a_collaborator?
```

Comment:
Correct me if I'm wrong:
We check the assigner is a collaborator and can indeed assign someone, that's right.
But then we check we can only assign collaborators.
So in the end this only allow collaborators to assign other collaborators, is this how it should be?